package.json

```diff
@@ -545,6 +553,7 @@
     "get-port": "^7.0.0",
     "json-schema-faker": "^0.5.8",
     "openapi-types": "^12.1.3",
+    "semver": "^7.7.1",
```

Comment:
why we need to upgrade this version

Reply:
Main porpose is to include it as production dependencies, and use latest version by the way.